src/containers/Layout/NotificationDrawerWrapper.tsx

```diff
@@ -1,42 +1,72 @@
 import {
+  Button,
   NotificationDrawer,
   NotificationDrawerBody,
   NotificationDrawerHeader,
   NotificationDrawerList,
   NotificationDrawerListItem,
   NotificationDrawerListItemBody,
-  NotificationDrawerListItemHeader
+  NotificationDrawerListItemHeader,
+  ButtonVariant
 } from '@patternfly/react-core';
-import React, { useContext } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { AppContext } from '../../context/context';
+import { TimesIcon, TimesCircleIcon } from "@patternfly/react-icons";
+import { NotificationActionTypes } from "../../context/actions/types";
+import { NotificationItem } from "../../context/reducers/notificationReducer";
 
 interface NotificationDrawerWrapperProps {
   onClose: () => void;
 }
 
 const NotificationDrawerWrapper: React.FC<NotificationDrawerWrapperProps> = ({ onClose }) => {
-  const { state: { notifications } } = useContext(AppContext);
+  const [disabled, setDisabled] = useState(true);
```

Comment:
Make the name of this variable more explicit that it is used for the Clear All button.